### PR TITLE
Fixes #680 changed the base URL

### LINKED ISSUE #680: Change default API URL to dev version
**Is it a ?**

- [ ] Bug Report

- [ ] Feature Request
    
- [x] Chore

### Chore

**Description**

Change the default dev API link from https://open-event-api.herokuapp.com to https://open-event-api-dev.herokuapp.com

The development branch of open event server is now hosted on the second link and thus, all new features and bug fixes will be present there

### PR DESCRIPTION
**Pull request by MLPJ**
Fixes #680 
Changes
changed the base URL to  https://open-event-api-dev.herokuapp.com

### CODE REVIEW

**Comment by MLPJ** [comment]
@iamareebjamal I did the change of URL you asked for but I can't relate it to the issue. How to do that

**Comment by iamareebjamal** [comment]
When you created the pull request, there was a template which was generated. Did you read it?

Write `Fixes #<issue_num>` at the top of pull request message

File: app/src/main/java/org/fossasia/openevent/app/common/Constants.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 public final class Constants {
     //url to login organiser
-    public static final String BASE_URL = "https://open-event-api.herokuapp.com/v1/";
+    public static final String BASE_URL = " https://open-event-api-dev.herokuapp.com/v1/";
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/common/Constants.java]: Extra space in URL

**Comment by iamareebjamal** [comment]
Also, regarding commit names, check older commit messages in the repo

**Comment by MLPJ** [comment]
@iamareebjamal  did the changes please take a look

**Comment by iamareebjamal** [comment]
You did not update the PR message

**Comment by MLPJ** [comment]
image
isn't this right?

**Comment by MLPJ** [comment]
@iamareebjamal  but still the issue is not set to this PR, is it because i'm not assigned to the issue?

**Comment by MLPJ** [comment]
@iamareebjamal Sorry I had updated the PR title. Now it is right

**Pull request merged**